gldt_active

jorgenatz opened the pull request:
gldt_active.sh is a script used to prune non-actively developed distributions from gldt.csv, avoiding parents of actively developed distributions. The modified build.sh can use this script to generate a separate gldt_active.csv and create a cladogram based on the data, by passing "active" as the second argument. Attached is an example active cladogram, along with the csv data used to generate it.

Current error checking is quite lacking. I would be open to ideas.

I apologize for anything I did incorrectly.

FabioLolix commented:
SVG option is half broken, produce the same result as not using it except for naming the folder (DIST_gldtSVG)


set gldt_active.sh as executable

in build.sh update DISTFILES=" to include gldt_active.sh

add gldt_active.csv and gldt_active.conf to .gitignore

gldt_active.conf is a copy of gldt.conf, can be avoided?

jorgenatz commented:
I have updated build.sh by making the svg and active arguments case-insensitive, thereby remedying the SVG build issue. I have also created a different variable for the .conf basename, avoiding the need to copy, and I have added gldt_active.sh to DISTFILES.

Is there anything else I missed?

FabioLolix commented:
Good :)
* set gldt_active.sh as executable

* add gldt_active.sh in main DISTFILES

Then resolve conflicts, squash in a single commit and rebase to the latest commit

jorgenatz commented:
Sorry, I had forgotten about this. It should be correct now, assuming I didn't miss anything more.

FabioLolix commented:
Will be superseded by #87 or something else

I have changed my mind about it from 2 years ago
